doc/user/content/sql/system-catalog/mz_internal.md

```diff
@@ -848,32 +847,35 @@ messages and additional metadata helpful for debugging.
 | `messages_received`      | [`uint8`]    | The number of messages the worker has received from the external system. Messages are counted in a source type-specific manner. Messages do not correspond directly to updates: some messages produce multiple updates, while other messages may be coalesced into a single update. |
 | `bytes_received`         | [`uint8`]    | The number of bytes the worker has read from the external system. Bytes are counted in a source type-specific manner and may or may not include protocol overhead.                                                                                                                  |
 | `updates_staged`         | [`uint8`]    | The number of updates (insertions plus deletions) the worker has written but not yet committed to the storage layer.                                                                                                                                                                |
 | `updates_committed`      | [`uint8`]    | The number of updates (insertions plus deletions) the worker has committed to the storage layer.                                                                                                                                                                                    |
-| `envelope_state_bytes`   | [`uint8`]    | The number of bytes stored in the source envelope state.                                                                       |
 | `envelope_state_records` | [`uint8`]    | The number of individual records stored in the source envelope state.                                                                                                                                                                                                               |
-| `rehydration_latency`    | [`interval`] | The amount of time it took for the worker to rehydrate the source envelope state. |
+| `envelope_state_bytes`   | [`uint8`]    | The number of bytes stored in the source envelope state.                                                                                                                                                                                                                            |
+| `rehydration_latency`    | [`interval`] | The amount of time it took for the worker to rehydrate the source envelope state.                                                                                                                                                                                                   |
+| `snapshot_total`         | [`uint8`]    | Not yet populated.                                                                                                                                                                                                                                                                  |
```

Comment:
I would see if the `{{< warn-if-unreleased v0.88 >}}` macro allows you to add documentation for these fields now, but with a warning that they're unreleased. The warning will automatically disappear when v0.88 is fully released!

Reply:
oh nice!

Reply:
was able to do this with 0.87, 0.88/89 arent listed in the releases yet